cmd/describe/machinepool/cmd_test.go

```diff
@@ -25,6 +25,7 @@ Desired replicas:                      0
 Current replicas:                      
 Instance type:                         m5.xlarge
 Labels:                                
+Tags:                                  
```

Comment:
Can you also add one test which has tags? So that if the output is changed for some reason it would be picked up